docs/source/src/python/user-guide/expressions/categoricals.py

```diff
@@ -130,7 +134,10 @@
     schema_overrides={"species": pl.Categorical},
 )
 
-bears = pl.concat([male_bears, female_bears], how="vertical")
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=CategoricalRemappingWarning)
```

Comment:
Oh, this is still needed? 😪

Reply:
Yep!
https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings